umbrella/router.go

```diff
@@ -105,7 +102,12 @@ func getFuncRoutingInfo(reqUrlRoot string, values url.Values, cmdReg *map[string
 	}
 
 	funcName := argsList[0]
-	funcRoutingInfo, funcRoutingInfoOk := funcReg[funcName]
+	confirmedFuncName, err := confirmFunctionIsRegistered(funcName, funcReg)
+	if err != nil {
+		return nil, err
+	}
+
+	funcRoutingInfo, funcRoutingInfoOk := funcReg[confirmedFuncName]
```

Comment:
This is redundant based on what confirmFunctionIsRegistered does already.

Reply:
If I remember, I was thinking that might be the case, but I tried to work within what you had already written rather than overwrite what you had done. I'll go get my head back in the code and try to make the adjustment.

Reply:
dev-culture-tip: Changes such as this, engineers will expect you either to just make the change (then report Done), or hear reasoning for why you're delaying it. For the latter, it may be because you have a multi-part refactor. If that's the case, you should outline in the review's comments what the parts are.

Reply:
That makes sense -- thanks. And "Done" -- I think. :)